dask_sql/context.py

```diff
@@ -845,10 +813,44 @@ def _get_ral(self, sql):
         logger.debug(f"_get_ral -> LogicalPlan: {rel}")
         logger.debug(f"Extracted relational algebra:\n {rel_string}")
 
+        return rel, rel_string
+
+    def _compute_table_from_rel(self, rel: "LogicalPlan", return_futures: bool = False):
```

Comment:
Should the default for `return_futures` here be `True` in line with c.sql?

Reply:
Yeah, should probably set this to `True`